spec/import/mods_transform_spec.rb

```diff
@@ -4,7 +4,13 @@
 
 RSpec.describe 'Transforming MODS files' do
   let(:provide) do
-    { 'identifier' => identifier, 'exhibit_slug' => slug, 'inst_id' => 'stanford' }
+    {
```

Comment:
I don't think this block is used now that you've moved the definition to line 33.